docs/guide/catalog/product_variants/setting_up_variants_from_external_source.md

```diff
@@ -56,20 +48,20 @@ This is the place where the factory decides which function it internally uses to
     It is good practice to configure which method should be used depending on e.g. the class identifier (Content Type).
     This is the default configuration:
 
-    ```
+    ``` yaml
     silver_eshop.default.catalog_factory.ses_category: createCatalogNode
     silver_eshop.default.catalog_factory.ses_product: createOrderableProductNode
     ```
 
-`createCatalogElement()` must do additional differentiation when to call the method `createOrderableProductNode()` and when `createVariantProductNode()`
-because in the below example (standard implementation), the variants are stored as Content items of the same Content Type as regular products (`ses_product`),.
-In the default implementation the factory calls the `createVariantProductNode()` method whenever [eZ Matrix with variants](../catalog_features/product_variants/product_variants.md) is filled.
+`createCatalogElement()` must additionally decide when to call the method `createOrderableProductNode()` and when `createVariantProductNode()`
+because in the example below (standard implementation), the variants are stored as Content items of the same Content Type as regular products (`ses_product`).
```

Comment:
```suggestion
because in the example below (standard implementation), the variants are stored, like regular products, in Product Content items (`ses_product`).
```